packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/ContractHelpers.kt

```diff
@@ -57,6 +58,7 @@ data class Contract(
         return when(type) {
             SupportedContractType.STRING -> forge.anExtendedAsciiString()
             SupportedContractType.MAP -> forge.exhaustiveAttributes()
+            SupportedContractType.LIST -> forge.aList(0) {}
```

Comment:
We should generate a list with actual context or just use `emptyList` here. I guess it should be non-empty list though.

Reply:
I'll use emptyList. I'm afraid of forging values since the caller could be looking for a List of any type. Since the contract tests don't look at the values anyway other than to cast them, an empty list should work fine.